app/src/androidTest/java/org/xbmc/kore/tests/mediaprovider/MediaProviderMusicTest.java

```diff
@@ -222,7 +222,6 @@ public void querySongWithArtistWithoutAlbumTest() {
         assertNotNull(cursor);
         assertEquals("cursor size ", 1, cursor.getCount());
         assertTrue(cursor.moveToFirst());
-        Utils.cursorToString(cursor);
```

Comment:
This file got moved int the latest PR, and this i guess that this isn't needed anymore.

Can you remove it from this PR?